components/job-orchestration/job_orchestration/executor/compress/compression_task.py

```diff
@@ -322,10 +322,21 @@ def run_clp(
         if last_archive_stats is not None and (
             None is stats or stats["id"] != last_archive_stats["id"]
         ):
+            archive_id = last_archive_stats["id"]
+            archive_path = archive_output_dir / archive_id
+            if StorageEngine.CLP_S == clp_storage_engine:
+                metadata_uploader_cmd = [
+                    str(clp_home / "bin" / "indexer"),
+                    "--db-config-file",
+                    str(db_config_file_path),
+                    "default",  # hardcode the table name for now
+                    archive_path,
+                ]
+                proc = subprocess.Popen(
+                    metadata_uploader_cmd, stdout=subprocess.DEVNULL, stderr=stderr_log_file
+                )
+                proc.wait()
```

Comment:
This is replacing the previous value of `proc`, right? It also seems like you could use `subprocess.run` instead?